modules/concom/pages/panes.inc

```diff
@@ -105,7 +105,17 @@ function meetings()
                 if ($nowDate == $checkDate) {
                     echo "<td class='CONCOM-checkin-now'><a href='index.php?Function=concom/signin&meeting=".$value['MeetingID']."'>Sign In Now!</a>";
                 } else {
-                    echo "<td>Not Checked In";
+                    echo "<td>Not Checked In -";
+                    $checkPending = check_meeting_pastrequest($value['MeetingID']);
+
+                    if ($checkPending) {
+                        echo "Attendance request pending";
```

Comment:
Would we please get a space at the beginning of this string.  "Not Checked In -Attendance request pending"  no space between the '-' and 'A' bugs me.

Reply:
Got it